packages/ListBox/src/components/Content/Content.js

```diff
@@ -46,20 +47,18 @@ export default function Content(props) {
   const { refListBoxContainer } = state;
 
   /* NOTE no idea what ROLE should be this div when the ListBox is INLINE */
-  /* eslint-disable jsx-a11y/no-static-element-interactions */
   if (state.isInline) {
     return (
-      <div
+      <ContentStyled
```

Comment:
this is only wrapping the inline style of the listbox, do this style only applies when its inline? @tristanjasper

Reply:
Ok Yeah it makes sense to be applied inline and not inline so will make sure it is

Reply:
Discussed with @jayde-acl  and @nahumzs for now this dotted style is only being applied to the listBox when it is inline